source/module_hamilt_lcao/hamilt_lcaodft/operator_lcao/deepks_lcao.cpp

```diff
@@ -497,7 +497,7 @@ void hamilt::DeePKS<hamilt::OperatorLCAO<TK, TR>>::cal_HR_IJR(const double* hr_i
 
 inline void get_h_delta_k(int ik, double*& h_delta_k)
 {
-    h_delta_k = GlobalC::ld.H_V_delta.data();
+    h_delta_k = GlobalC::ld.H_V_delta[0].data();
```

Comment:
explain the meaning of '0', otherwise it is a magic number

Reply:
To make ld.H_V_delta (used for gamma-only) and ld.H_V_delta_k (used for multi-k) consistent, I add an additional dimension which will be convenient for future refactors. 
 Here 0 is caused by this.